static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -361,10 +357,11 @@ function AppStoreConnect({
           data: {sessionContext},
         }
       );
-
+      setUseSms(true);
```

Comment:
In the 2nd commit I changed the python code to entirely ignore this flag from the client, we already get this state via the sessionContext anyway.  So we could decide to remove it entirely out of the UI code and also no longer accept it on the backend API?

Reply:
anyway, one bug was that this was set regardless of whether the API call succeeded.  hence i move this.

Reply:
is `useSms` something that may be included in the response payload? is it possible for the request to be successful and the response to contain `useSms: false` with some sort of hint pointing you to a trusted device?

Reply:
> anyway, one bug was that this was set regardless of whether the API call succeeded. hence i move this.

this is strange..did you manage to reproduce the bug?

Reply:
user hits sms endpoint, sms endpoint returns error, client has useSms set but it actually isn't.  no not fully reproduced, but i understand how this can occur.